raitools/setup.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import os
+import setuptools
+
+# Version will be read from version.py
+version = ''
+name = 'raitools'
+# Fetch Version
+with open(os.path.join(name, '__version__.py')) as f:
+    code = compile(f.read(), f.name, 'exec')
+    exec(code)
+
+# Fetch ReadMe
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+# Use requirements.txt to set the install_requires
+with open('requirements.txt') as f:
+    install_requires = [line.strip() for line in f]
+
+setuptools.setup(
+    name=name,  # noqa: F821
+    version=version,  # noqa: F821
+    author="Roman Lutz, Ilya Matiach, Ke Xu",
+    author_email="raiwidgets-maintain@microsoft.com",
+    description="SDK API to assess fairness, explain "
+                "models, and analyze errors in Machine Learning",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/microsoft/responsible-ai-widgets",
+    packages=setuptools.find_packages(),
+    python_requires='>=3.6',
```

Comment:
Are we leaving this as >=3.6 for now? Or should we stick on >=3.5 for a bit?

Reply:
leaving it for >=3.6 since raiwidgets is also >=3.6